net/siproxd/src/opnsense/mvc/app/controllers/OPNsense/Siproxd/GeneralController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+    Copyright (C) 2017 Michael Muenz
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    1. Redistributions of source code must retain the above copyright notice,
+       this list of conditions and the following disclaimer.
+
+    2. Redistributions in binary form must reproduce the above copyright
+       notice, this list of conditions and the following disclaimer in the
+       documentation and/or other materials provided with the distribution.
+
+    THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+    AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+    OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+
+namespace OPNsense\Siproxd;
+
+class GeneralController extends \OPNsense\Base\IndexController
+{
+    public function indexAction()
+    {
+        $this->view->title = gettext("Siproxd settings");
```

Comment:
not an requirement but maybe upcase the S in Settings?

Reply:
Hm, I'm cloning always from the latest plugins, so far it was never requested to change it to upper case. Doing a fgrep in plugins results in a mix of upper and lower.

Reply:
the command `git grep "view->title" | cut -d "=" -f 2` tells something else...

![spectacle nn3815](https://user-images.githubusercontent.com/3713914/30744985-b82940ec-9fa4-11e7-8fb8-2f0b556e5d5c.png)

Reply:
@fraenki the comand reveals that your controllers are missing `gettext` calls for translation.

Reply:
I just searched in plugins, there it's quite balanced:

$this->view->title = gettext("FreeRADIUS Settings");
<?=gettext("L2TP settings"); ?></b></td>
<?=gettext("PPTP settings"); ?></b></td>
$this->view->title = gettext("BGP Settings");
$this->view->title = gettext("Routing-Settings");
$this->view->title = gettext("IS-IS-Settings");
$this->view->title = gettext("OSPFv3 Settings");
$this->view->title = gettext("OSPF Settings");
$this->view->title = gettext("RIP Settings");
<?=gettext("Global settings"); ?></strong>
<?=gettext("UPnP and NAT-PMP Settings");?></strong>
<?= gettext('This service can be used to wake up (power on) computers by sending special "Magic Packets". The NIC in the computer that is to be woken up must support Wake on LAN and has to be configured properly (WOL cable, BIOS settings).');?>
$this->view->title = gettext("Siproxd settings");
$this->view->title = gettext("Siproxd settings");
$this->view->title = gettext("Collectd settings");
$this->view->title = gettext('Zabbix Agent Settings');
$this->view->title = gettext("ClamAV settings");
$this->view->title = gettext('Monit System Monitoring - Settings');
$this->view->title = gettext("C-ICAP Settings");

Reply:
@mimugmail for me this looks like there is a difference between the MVC plugins and the legacy plugins.

Maybe @fichtner or @AdSchellevis has a coding convention here.

Reply:
Uppercase for titles is preferred but I wouldn't bikeshed over this :)

Reply:
Ok, I'll try to make the others upper case too when this is through :)